install/helm/agones/templates/crds/_gameserverspecvalidation.yaml

```diff
@@ -148,4 +148,32 @@ properties:
             type: integer
             minimum: 1
             maximum: 2147483648
+      alpha:
+        type: object
+        title: Alpha properties for the GameServer
+        properties:
+          players:
+            type: object
+            title: Configuration of player capacity
+            properties:
+              initialCapacity:
+                type: integer
+                title: The intial player capacity that this Game Server has
```

Comment:
nit: I was always told not to end sentences with 'has'. Maybe rephrase as 'The initial player capacity of this Game Server'

Reply:
Words good! :+1: